BookingsApi/BookingsApi.IntegrationTests/Api/V1/Hearings/GetHearingsForNotificationTest.cs

```diff
@@ -0,0 +1,19 @@
+namespace BookingsApi.IntegrationTests.Api.V1.Hearings;
+
+public class GetHearingsForNotificationTest: ApiTest
+{
+    [Test]
+    public async Task should_return_ok()
+    {
+        // arrange
```

Comment:
What is this testing? There are no hearings, no assertions on the response model. This needs to make sure the api is protected from future accidental breaks

Reply:
it is working progress

Reply:
```
// arrange
Single day hearing before the cutoff
Single day hearing after the cutoff
Single day hearing within the cutoff
Multi day hearing before the cutoff
multi day hearing after the cutoff
Multi day hearing within the cutoff

// act
call the api

// assert
check the response contains single day hearing within cutoff and total days is 1
check the response contains multi day hearing within cutoff and total days is however many you created in the setup
```

this should provide enough confirm the query pulls in the correct data

Reply:
done